doofinder.php

```diff
@@ -1681,4 +1690,14 @@ private function displayGeneralMsg($string, $type, $alert, $link = false)
 
         return $this->context->smarty->fetch($this->local_path . 'views/templates/admin/display_msg.tpl');
     }
+
+    private function searchLayerMustBeInitialized()
+    {
+        $displayMobile = Configuration::get('DF_SHOW_LAYER_MOBILE', true);
+        $displayDesktop = Configuration::get('DF_SHOW_LAYER', true);
+        $isMobile = Context::getContext()->isMobile();
+
+        return (!empty($isMobile) && $displayMobile) ||
+        (empty($isMobile) && $displayDesktop);
```

Comment:
```suggestion
        return ($isMobile && $displayMobile) || (!$isMobile && $displayDesktop);
```
I think this is more readable